packages/user/Fractals/plugin/wit/world.wit

```diff
@@ -5,6 +5,7 @@ interface admin-fractal {
 
     create-fractal: func(fractal-account: string, guild-account: string, name: string, mission: string) -> result<_, error>;
     exile-member: func(member: string) -> result<_, error>;
+    set-dist-interval: func(distribution-interval: u32) -> result<_, error>;
```

Comment:
Consider naming `distribution-interval-sec` if indeed the unit is `seconds`. Alternatively (or in addition to), doc comments